docs/integrations/sources/intercom.md

```diff
@@ -49,6 +49,7 @@ The Intercom connector should not run into Intercom API limitations under normal
 
 | Version | Date       | Pull Request                                             | Subject                                                                                       |
 |:--------| :--------- | :------------------------------------------------------- | :-------------------------------------------------------------------------------------------- |
+| 0.3.0   | 2023-05-25 | [29598](https://github.com/airbytehq/airbyte/pull/29598) | Migrate to latest cdk==0.50.2                                                                 |
```

Comment:
Let's change subject to more human readable.

Reply:
Not sure what else I can add, so added some words describing why migration even take place

Reply:
how about:
```
"Resolved conflicts after switching to the latest CDK version."
```

Reply:
I have already prepared something like:
`Update custom components to make them compatible with latest cdk version 0.50.2, simplify logic, update schemas`

Reply:
I don't think we should include the specific version in the description, knowing that we should always use `latest` one.

Reply:
Agree. I will remove version

Reply:
Done